processing-status-notifications-fa/src/main/kotlin/gov/cdc/ocio/functions/http/SubscribeWebsocketNotifications.kt

```diff
@@ -0,0 +1,81 @@
+package gov.cdc.ocio.functions.http
+
+import com.microsoft.azure.functions.HttpRequestMessage
+import com.microsoft.azure.functions.HttpStatus
+import java.util.*
+import com.microsoft.azure.functions.HttpMethod
+import com.microsoft.azure.functions.HttpResponseMessage
+import com.microsoft.azure.functions.annotation.AuthorizationLevel
+import com.microsoft.azure.functions.annotation.HttpTrigger
+import gov.cdc.ocio.cache.InMemoryCacheService
+import gov.cdc.ocio.model.http.SubscriptionResult
+import gov.cdc.ocio.model.http.SubscriptionType
+import mu.KotlinLogging
+import java.time.Instant
+
+
+class SubscribeWebsocketNotifications(
+    private val request: HttpRequestMessage<Optional<String>>) {
+    private val logger = KotlinLogging.logger {}
+    private val cacheService: InMemoryCacheService = InMemoryCacheService()
+
+    fun run(
+        @HttpTrigger(name="req",
```

Comment:
You don't need the HttpTrigger annotation here again, this is in your wrapper that calls this.

Reply:
done